website/www/site/content/en/_index.md

```diff
@@ -1 +1,13 @@
-**Home**
\ No newline at end of file
+<!--
```

Comment:
Please make it a part of previous PR

Reply:
Also, you could already put these metadata stuff (title etc) here

Reply:
Just skip this for now. I've already made some changes to the next PR for the main page. 😛